app/controllers/feature_flags_controller.rb

```diff
@@ -1,13 +0,0 @@
-class FeatureFlagsController < ApplicationController
```

Comment:
We may need to bring this back later but I thought I'd remove it until I spoke to you about libraries that have this kind of functionality built-in.